osi_motionrequest.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_common.proto";
+import "osi_version.proto";
+
+package osi3;
+
+//
+// \brief This message is intended as an interface between a
+// highly-automated-driving function (HAD function) and the actuator management.
+//
+// The HAD function can either send a desired future trajectory or a desired
+// future state. The message can be defined by an additional variable.
+//
+// \note The coordinate system is defined as right-handed.
+// All coordinates and orientations are relative to the global coordinate system.
+// The reference point of the vehicle is the center of the 3D bounding box of the vehicle.
+// Units are m for positions, m/s for velocities, and m/s^2 for accelerations.
+//
+message MotionRequest
+{
+    // The interface version used by the sender (simulation environment).
+    //
+    optional InterfaceVersion version = 1;
+
+    // The data timestamp of the simulation environment.
+    // A reference to \c Timestamp message.
+    //
+    optional Timestamp timestamp = 2;
+
+    // Define the option that is used to specify the motion request.
+    // This must be set. Additionally, the field corresponding to the specified 
+    // option must be set.
+    //
+    optional OutputOption output_option = 3;
+
+    // Defines a desired state.
+    // If the output option is set to DESIRED_STATE, this field must be set.
+    //
+    optional DesiredState desired_state = 4;
+
+    // Defines a desired trajectory.
+    // If the output option is set to DESIRED_TRAJECTORY, this field must be set.
+    //
+    optional DesiredTrajectory desired_trajectory = 5;
+
+    // Define different options for HAD function output.
+    // Each option corresponds to a field in the message.
+    //
+    enum OutputOption
+    {
+        // Desired state calculated by the HAD function.
+        //
+        OUTPUT_OPTIONS_DESIRED_STATE = 0;
+
+        // Desired trajectory calculated by the HAD function.
+        //
+        OUTPUT_OPTIONS_TRAJECTORY = 1;
+    }
+
+    // \brief The desired state is calculated by the HAD function as a result of
+    // the motion planning stack.
+    //
+    // The actuator management is supposed to reach the desired state at the
+    // specified time.
+    //
+    message DesiredState
+    {
+        // A reference to \c Timestamp message.
+        //
+        optional Timestamp timestamp = 1;
```

Comment:
This is not the current timestamp but the time when the desired state should be reached, right?